exec/java-exec/src/main/java/org/apache/drill/exec/server/Drillbit.java

```diff
@@ -283,6 +284,7 @@ public static Drillbit start(final DrillConfig config, final RemoteServiceSet re
     try {
       bit.run();
     } catch (final Exception e) {
+      logger.error("Failure during initial startup of Drillbit.\n[{}]", ErrorHelper.buildCausesMessage(e));
```

Comment:
I'd suggest using `logger.error("Failure during initial startup of Drillbit.", e);` Log parsers like Lilith use the exception field, and changes to ErrorHelper aren't required.